cycle/docker-compose.yaml

```diff
@@ -886,34 +885,21 @@ services:
         ts_hexrotcomm: ${ts_hexrotcomm}
         ts_tcpip: ${ts_tcpip}
 
-  bare-centos-community:
-    image: lsstts/centos7:${CYCLE}
-    build:
-      context: ../build/
-      dockerfile: ./develop-env/bare-centos/Dockerfile
-      labels:
-        com.description: "LSST Telescope and Site base CentOS 7 image for cycle ${CYCLE}."
-      args:
-        dds_community_version: ${dds_community_version}
-        dds_community_build: ${dds_community_build_el7}
-        ts_xml: ${ts_xml}
-        ts_sal: ${ts_sal}
-
   conda-builder-aarch64-community:
     image: lsstts/conda_package_builder_aarch64:${CYCLE}
     build:
       context: ../build/
       dockerfile: ./conda_builder/aarch64/Dockerfile
       labels:
-        com.description: "LSST Telescope and Site base AlmaLinux 8 aarch64 image for cycle ${CYCLE}. This image doesn't contain OpenSpliceDDS."
+        com.description: "LSST Telescope and Site base AlmaLinux 8 aarch64 image for cycle ${CYCLE} without OpenSpliceDDS."
 
   conda-builder-aarch64-private:
     image: ${hub}/conda_package_builder_aarch64:${CYCLE}
     build:
       context: ../build/
       dockerfile: ./conda_builder/aarch64/Dockerfile
       labels:
-        com.description: "LSST Telescope and Site base AlmaLinux 8 aarch64 image for cycle ${CYCLE}. This image doesn't contain OpenSpliceDDS."
+        com.description: "LSST Telescope and Site base AlmaLinux 8 aarch64 image for cycle ${CYCLE} without OpenSpliceDDS."
```

Comment:
If this is a private build, doesn't it have OpenSplice in it?

Reply:
Indeed, the image does not have OpenSplice in it so probably no reason to make it private. We can talk about making it public in the future.